libs/portals/admin/document-provider/src/lib/utils.ts

```diff
@@ -0,0 +1,25 @@
+// Utility for formatting numbers in Icelandic locale
```

Comment:
I think using Intl.NumberFormat would be beneficial here, cleaner and no need for manual "K"/"M" handling.